images/win/scripts/Installers/Validate-Git.ps1

```diff
@@ -26,24 +29,35 @@ if( $(git-lfs version) -match  'git-lfs\/(?<version>.*) \(Git.*' )
 }
 
 # Adding description of the software to Markdown
-$SoftwareName = "Git"
-
-$Description = @"
+$GitSoftware = @(
+    [pscustomobject]@{
+        SoftwareName = "Git";
+        Description = @"
 _Version:_ $gitVersion<br/>
 _Environment:_
 * PATH: contains location of git.exe
 "@
-
-Add-SoftwareDetailsToMarkdown -SoftwareName $SoftwareName -DescriptionMarkdown $Description
-
-# Adding description of the software to Markdown
-$SoftwareName = "Git Large File Storage (LFS)"
-
-$Description = @"
+    },
+    [pscustomobject]@{
+        SoftwareName = "Git Large File Storage (LFS)";
+        Description = @"
 _Version:_ $gitLfsVersion<br/>
 _Environment:_
 * PATH: contains location of git-lfs.exe
 * GIT_LFS_PATH: location of git-lfs.exe
 "@
+    },
+    [pscustomobject]@{
+        SoftwareName = "Hub CLI";
+        Description = @"
+_Version:_ $hubVersion<br/>
+_Environment:_
+* PATH: contains location of hub.exe
+* GIT_LFS_PATH: location of hub.exe
+"@
+    }
+)
```

Comment:
This section for my mind contains very difficult formatting style. Shorter variant:

```
# Adding description of the software to Markdown
$GitSoftwareDescription = @"
_Version:_ $gitVersion<br/>
_Environment:_
* PATH: contains location of git.exe
"@

Add-SoftwareDetailsToMarkdown -SoftwareName "Git" -DescriptionMarkdown $GitSoftwareDescription

$GitLFSDescription = @"
_Version:_ $gitLfsVersion<br/>
_Environment:_
* PATH: contains location of git-lfs.exe
* GIT_LFS_PATH: location of git-lfs.exe
"@

Add-SoftwareDetailsToMarkdown -SoftwareName "Git Large File Storage (LFS)" -DescriptionMarkdown $GitLFSDescription

$HubCliDescription = @"
_Version:_ $hubVersion<br/>
_Environment:_
* PATH: contains location of hub.exe
* GIT_LFS_PATH: location of hub.exe
"@

Add-SoftwareDetailsToMarkdown -SoftwareName "Hub CLI" -DescriptionMarkdown $HubCliDescription

```

Reply:
It's  copy-past. I don't think that's right

Reply:
Not in this case. It's longer and very poor formatted style.

Reply:
https://github.com/andy-mishechkin/virtual-environments/commit/9f7ab8176cb072b05ff08150f8fd8221a3cdfb52